README.md

```diff
@@ -144,6 +146,26 @@ $ sudo dokcer compose stop
 
 ## FAQ
 
+### Use Ascender without Docker
+
+We recommend using Ascender with Docker as described above. However, you might not be able to install Docker in your development environment due to permission issues or etc.
```

Comment:
Does `described above` mean `- **Container**: Use of [Docker](https://www.docker.com/) reduces development environment dependencies and improves code portability.`?

Reply:
Yes, it is correct. And it is also explained in [the slide format resources about Ascender (Japanese)](https://cvpaperchallenge.github.io/Britannica/ascender/ja).